cirq-core/cirq/ops/measurement_gate.py

```diff
@@ -91,6 +91,19 @@ def __init__(
         self._invert_mask = invert_mask or ()
         if self.invert_mask is not None and len(self.invert_mask) > self.num_qubits():
             raise ValueError('len(invert_mask) > num_qubits')
+
+        def right_trim_mask(invert_mask):
+            if invert_mask is None or len(invert_mask) == 0:
+                return invert_mask
```

Comment:
this assumes that `invert_mask` can be `None` which it's not
https://github.com/quantumlib/Cirq/blob/d33b1a71ac9721e762a2fa1b9e9871edfe187a3c/cirq-core/cirq/ops/measurement_gate.py#L52